build.gradle

```diff
@@ -442,7 +445,8 @@ dependencies {
 	productionRuntimeServer "net.fabricmc:fabric-installer:${project.installer_version}:server"
 }
 
-import net.fabricmc.loom.util.Platform
+
+import org.kohsuke.github.GHReleaseBuilder
```

Comment:
No need to move these imports, I think its generally better to keep the imports close to where they are used.